lang/vi/spec/v2.0.0.md

```diff
@@ -0,0 +1,106 @@
+---
+Tiêu đề: Ý nghĩa việc đánh số phiên bản 2.0.0
+Ngôn ngữ: Tiếng Việt
+---
+
+Đánh số phiên bản 2.0.0
+===
+
+Tổng quan
+---
+Đưa ra một cấu trúc phiên bản MAJOR.MINOR.PATCH, gia tăng như thế này:
+1. Số phiên bản MAJOR khi bạn có các thay đổi lớn về API
+2. Số phiên bản MINOR khi bạn thêm chức năng tương thích ngược với phiên bản trước
+3. Số phiên bản PATCH khi bạn làm một bản vá lỗi tương thích ngược với phiên bản trước
+
+Bổ sung các nhãn cho tiền phát hành và xây dựng các siêu dữ liệu sẵn sàng như là một tiện ích mở rộng theo định dạng MAJOR.MINOR.PATCH
+
+Giới thiệu
+---
+
+Trong thế giới của quản lý phần mềm, ở đó tồn tại một nơi được gọi là "địa ngục của sự phụ thuộc". Hệ thống của bạn ngày càng lớn hơn và bạn càng tích hợp nhiều gói vào trong phần mềm của mình hơn, Một ngày nào đó bạn sẽ tuyệt vọng tìm kiếm bản thân mình trong cái địa ngục này.
+
+Trong các hệ thống có nhiều sự phụ thuộc, phát hành một phiên bản mới có thể nhanh chóng trở thành một cơn ác mộng. Nếu các thông số kỹ thuật phụ thuộc quá chặt chẽ với nhau, bạn đang ở trong sự nguy hiểm của khóa phiên bản (không có khả năng nâng cấp một gói mà không phải phát hành các phiên bản mới của mỗi gói phụ thuộc). Nếu các phụ thuộc được chỉ định rất lỏng lẻo, bạn chắc chắn sẽ ít mắc phải sự linh tinh phiên bản (giả định khả năng tương thích với nhiều phiên bản trong tương lai là hợp lý). Địa ngục của sự phụ thuộc là khi khóa phiên bản hoặc sự linh tinh của phiên bản ngăn cản bạn dễ dàng và an toàn chuyển tiếp dự án của mình.
```

Comment:
`sự linh tinh phiên bản`
=> `sự lộn xộn của phiên bản`

Reply:
`an toàn chuyển tiếp dự án của mình` => `chuyển tiếp dự án của mình một cách an toàn`